forum/settings.py

```diff
@@ -54,7 +55,14 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
 ]
+CORS_ALLOW_CREDENTIALS = True
+CORS_ORIGIN_WHITELIST = (
+    'http://localhost:3000',
+    '0.0.0.0',
```

Comment:
Having this, all other entries are redundant. Though, it's very unsafe.

Reply:
I don't understand what is redundant? CORS_ALLOW_CREDENTIALS or CORS_ORIGIN_WHITELIST? Do I have to delete something?